web/components/FcDrawerViewCollisionReports.vue

```diff
@@ -0,0 +1,304 @@
+<template>
+  <div class="fc-drawer-view-collision-reports d-flex flex-column">
+    <FcDialogConfirm
+      v-model="showConfirmLeave"
+      textCancel="Stay on this page"
+      textOk="Leave"
+      title="Leave Reports"
+      @action-ok="actionLeave">
+      <span class="body-1">
```

Comment:
for future work: make sure the location search bar collapses here as well as in the original View Reports page